src/vcpkg/commands.export.cpp

```diff
@@ -438,11 +438,6 @@ namespace vcpkg::Export
                 return parse_package_spec(
                     arg, default_triplet, default_triplet_used, COMMAND_STRUCTURE.get_example_text());
```

Comment:
Should the usage of 
`default_triplet, default_triplet_used`
generally be removed? 
I kind of let them in since removing then would create a large changeset.